src/commands/dev/watch.rs

```diff
@@ -0,0 +1,33 @@
+use std::mem;
+use std::sync::{mpsc, Arc, Mutex};
+
+use crate::commands;
+use crate::commands::dev::get_preview_id;
+use crate::commands::dev::server_config::ServerConfig;
+
+use crate::settings::global_user::GlobalUser;
+use crate::settings::toml::Target;
+
+pub fn watch_for_changes(
+    target: Target,
+    user: Option<GlobalUser>,
+    server_config: &ServerConfig,
+    preview_id: Arc<Mutex<String>>,
+    session_id: &str,
+    verbose: bool,
+) -> Result<(), failure::Error> {
+    let (tx, rx) = mpsc::channel();
```

Comment:
what are tx and rx? we really gotta go for readability with code like this, and comments, even if it's just like "this is the boilerplate from the getting started for this package.

Reply:
this is standard notation for mpsc channels, but i'll change it to `sender and receiver` since it's more readable 😄